README.md

```diff
@@ -17,5 +17,6 @@ Scaffolded with t3.gg
 # Contributing
 
 You will likely need to make a clerk account to get the .env secret for this to test locally. [Clerk env docs](https://clerk.com/docs/nextjs/set-environment-keys)
-You'll also need to make a free mySQL database with [Planetscale](https://planetscale.com/). Once you create the database, select 'connect' and choose 'Connect with Prisma'. Take the generated url and place it in your .env.
+You'll also need to make a free mySQL database with [Planetscale](https://planetscale.com/). Once you create the database, select 'connect' and choose 'Connect with Prisma'. Take the generated url and place it in your .env. Then run "npm run postinstall" and "npx prisma db push".
```

Comment:
What does the npm run postinstall do?

Reply:
I think postinall runs after npm install, we should just have them run that command afterwards.